### DISCUSSION TITLE
some parameters not working

### DISCUSSION THREAD

**Discussion by ghost** [davatorium/rofi#1555] [Theming] [answered]
in version 1.7+, some parameters don't work, for example: line, width, padding, column. however, it can be configured with a theme. so, how can i run rofi (like use line/width) with parameters to make rofi dynamically?

ex: `rofi -show drun -l 10`

**Comment by DaveDavenport** [comment]
Can I suggest the [search](https://github.com/davatorium/rofi/discussions?discussions_q=lines) function.

**Comment by ghost** [comment]
Edit in Tuesday, 06 September 2020 :

I'm just want to update this thread to avoid misunderstanding.

> Can I suggest the [search](https://github.com/davatorium/rofi/discussions?discussions_q=lines) function.

and next thread

> How can i exec the rofi width this code?

The answer to that suggestion in here https://github.com/davatorium/rofi/discussions/1555#discussioncomment-1850101

Maybe it will create misunderstandings to others, which it make me don't read it and send a reply without reading it.

So..., actually i have read that https://github.com/davatorium/rofi/discussions/1555#discussioncomment-1845668, buat i forgot to answer.

Thanks for your attention.

---

How can i exec the rofi width this code?

```
bindsym $super+d exec rofi -theme-str 'listview {lines: 5;}' -theme-str 'window {width: 1000px;}' -show run
```
it gives error

![Screenshot_Tue-21Dec21_06 45](https://user-images.githubusercontent.com/70983461/146847261-b34149e5-bd71-4a18-8a8a-21952b60fe99.png)

but it work in terminal

![Screenshot_Tue-21Dec21_06 50](https://user-images.githubusercontent.com/70983461/146847610-79ceacc7-813e-49b1-bf43-c3640f992721.png)
![Screenshot_Tue-21Dec21_06 57_000](https://user-images.githubusercontent.com/70983461/146848172-70dfe919-59eb-4cd9-b920-947a4e52f31d.png)

**Reply by ghost** [reply]
Reference : https://github.com/davatorium/rofi/discussions/1455#discussioncomment-1306043

**Reply by ghost** [reply]
I have tried many ways to fix it and got nothing

**Reply by TonCherAmi** [reply]
This is not a rofi issue.

You need to surround the command you want to bind with quotes (`"`):

```
bindsym $super+d exec "rofi -theme-str 'listview {lines: 5;}' -theme-str 'window {width: 1000px;}' -show run"
```

**Reply by DaveDavenport** [reply]
What is bindsym?
I am missing a whole lot of context? but it looks like your issue is with command escaping in an unnamed other tool, not rofi.

**Reply by TonCherAmi** [reply]
It's an i3 thing :)

https://i3wm.org/docs/userguide.html#keybindings

**Reply by ghost** [reply]
> but it looks like your issue is with command escaping in an unnamed other tool

> You need to surround the command you want to bind with quotes (")

Thanks, it worked now... and I forgot that the command can be run with quotes ( " )

![Screenshot_Tue-21Dec21_19 01_000](https://user-images.githubusercontent.com/70983461/146926742-60a66681-6156-4dae-bb65-93e41f81701d.png)

![Screenshot_Tue-21Dec21_19 01](https://user-images.githubusercontent.com/70983461/146926727-9dc34d54-7387-4613-8ee4-a59f9684fdc2.png)

**Comment by ghost** [comment, marked as answer]
https://github.com/davatorium/rofi/discussions/1555#discussioncomment-1850672

**Reply by ghost** [reply]
Ok... many thanks to all
